app/docker/networks/edit/NetworkContainersTable.tsx

```diff
@@ -0,0 +1,84 @@
+import { Widget, WidgetBody, WidgetTitle } from '@/portainer/components/widget';
+import { DetailsTable } from '@/portainer/components/DetailsTable';
+import { Button } from '@/portainer/components/Button';
+import { Authorized } from '@/portainer/hooks/useUser';
+import { EnvironmentId } from '@/portainer/environments/types';
+import { Link } from '@/portainer/components/Link';
+
+import { NetworkContainers, NetworkId } from '../types';
+import { useDisconnectContainer } from '../queries';
+
+type Props = {
+  containers: NetworkContainers;
+  nodeName: string;
+  environmentId: EnvironmentId;
+  networkId: NetworkId;
```

Comment:
@chiptus is it better to pass in the envId and networkId and handle disconnectContainer.mutate({ containerId }) in this component,

Or should I pass in disconnectContainer.mutate({ containerId })  from NetworkDetailsView, so that I don't need to pass envId and networkId?